.github/workflows/test.yml

```diff
@@ -15,47 +25,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 7.4
```

Comment:
removed the two other jobs, as they were the same and could be replaced with a matrix